spec/controllers/requests_controller_spec.rb

```diff
@@ -38,7 +38,7 @@
     describe "GET #index" do
       it "assigns all requests as @requests" do
         get :index, session: valid_session
-        expect(assigns(:requests)).to eq([@a_request])
+        expect(response).to redirect_to(root_url)
```

Comment:
Looking at the comment for `valid_session` ("This should return the minimal set of values that should be in the session in order to pass any filters (e.g. authentication)") I would say this test shouldn't have been changed, but the session should have been adapted. If adapting the session seems overly tedious (which seems like it could be the case looking at [this](https://github.com/plataformatec/devise/wiki/How-To:-Test-controllers-with-Rails-3-and-4-(and-RSpec)#rails-3-rspec-scaffolds)), I guess changing the title of the test case could be sufficient as well.